jdbc/src/test/java/software/amazon/timestream/jdbc/TimestreamDatabaseMetaDataTest.java

```diff
@@ -52,6 +52,59 @@ void init() throws SQLException {
     dbMetaData = new TimestreamDatabaseMetaData(mockConnection);
   }
 
+  /**
```

Comment:
Are there new tests added to test the code changes done in [TimestreamTablesResultSet.java](https://github.com/awslabs/amazon-timestream-driver-jdbc/pull/22/commits/85237b92ca72aec9d79ba98b4d1207bb86d3fb2c#diff-fa970402ed9dc285046d53eb324502a6fab375cbd21380f1734d0710af1e2a38) ?